src/jdk.httpserver/share/classes/com/sun/net/httpserver/Headers.java

```diff
@@ -36,43 +41,48 @@
  * the value associated with each key is
  * a {@link List}{@literal <}{@link String}{@literal >} with one
  * element for each occurrence of the header name in the request or response.
- * <p>
- * For example, if a response header instance contains
+ *
+ * <p> For example, if a response header instance contains
  * one key "HeaderName" with two values "value1 and value2"
  * then this object is output as two header lines:
+ *
  * <blockquote><pre>
  * HeaderName: value1
  * HeaderName: value2
  * </pre></blockquote>
- * <p>
- * All the normal {@link java.util.Map} methods are provided, but the following
+ *
```

Comment:
The lines in the list look very long. Might be better to reformat to keep lines less than 80 chars (or whatever the limit is supposed to be).

Reply:
Change made in commit 4de7e6c